<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="menu_File_Open"/>
    <addaction name="menu_File_FolderOpen"/>
    <addaction name="separator"/>
    <addaction name="menu_File_Close"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>表示</string>
    </property>
    <addaction name="menu_View_ScrollHand"/>
    <addaction name="separator"/>
    <addaction name="menu_View_FullSize"/>
    <addaction name="menu_View_FitWindow"/>
    <addaction name="menu_View_FitImage"/>
    <addaction name="menu_View_SetScale"/>
    <addaction name="separator"/>
    <addaction name="menu_View_Rotate90"/>
    <addaction name="menu_View_Rotate180"/>
    <addaction name="menu_View_Rotate270"/>
    <addaction name="menu_View_SetRotate"/>
    <addaction name="separator"/>
    <addaction name="menu_View_FullScreen"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>ヘルプ</string>
    </property>
    <addaction name="menu_Help_Version"/>
   </widget>
   <widget class="QMenu" name="menu_Filter">
    <property name="title">
     <string>フィルタ</string>
    </property>
    <addaction name="menu_Filter_Monotone"/>
    <addaction name="menu_Filter_Sharpness"/>
    <addaction name="menu_Filter_Binarized"/>
    <addaction name="menu_Filter_Averagin"/>
    <addaction name="separator"/>
    <addaction name="menu_Filter_Antialiasing"/>
   </widget>
   <widget class="QMenu" name="menu_Slideshow">
    <property name="title">
     <string>スライドショー</string>
    </property>
    <addaction name="menu_Slideshow_Slideshow"/>
    <addaction name="menu_Slideshow_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>ウィンドウ</string>
    </property>
    <addaction name="menu_Window_Thumbnail"/>
    <addaction name="menu_Window_List"/>
    <addaction name="separator"/>
    <addaction name="menu_Window_Histogram"/>
    <addaction name="menu_Window_ToneCurve"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Filter"/>
   <addaction name="menu_Slideshow"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="menu_File_Open">
   <property name="text">
    <string>ファイルを開く</string>
   </property>
  </action>
  <action name="menu_View_FullSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>原寸大で表示する</string>
   </property>
  </action>
  <action name="menu_View_FitWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ウィンドウに合わせて表示する</string>
   </property>
  </action>
  <action name="menu_Help_Version">
   <property name="text">
    <string>バージョン</string>
   </property>
  </action>
  <action name="menu_File_FolderOpen">
   <property name="text">
    <string>フォルダを開く</string>
   </property>
  </action>
  <action name="menu_File_Close">
   <property name="text">
    <string>ファイルを閉じる</string>
   </property>
  </action>
  <action name="menu_View_SetScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>倍率を指定して表示する</string>
   </property>
  </action>
  <action name="menu_View_FullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>フルスクリーン</string>
   </property>
  </action>
  <action name="menu_Slideshow_Slideshow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>スライドショー</string>
   </property>
  </action>
  <action name="menu_Slideshow_Settings">
   <property name="text">
    <string>スライドショーの設定</string>
   </property>
  </action>
  <action name="menu_Filter_Monotone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>モノクロ</string>
   </property>
  </action>
  <action name="menu_Filter_Sharpness">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>鮮鋭化</string>
   </property>
  </action>
  <action name="menu_Window_List">
   <property name="text">
    <string>プレイリスト</string>
   </property>
   <property name="toolTip">
    <string>プレイリスト</string>
   </property>
  </action>
  <action name="menu_Window_Histogram">
   <property name="text">
    <string>ヒストグラム</string>
   </property>
  </action>
  <action name="menu_View_Rotate90">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>時計回りに90度回転</string>
   </property>
  </action>
  <action name="menu_View_Rotate180">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>時計回りに180度回転</string>
   </property>
  </action>
  <action name="menu_View_Rotate270">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>時計回りに270度回転</string>
   </property>
  </action>
  <action name="menu_View_SetRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>回転角度を指定して表示する</string>
   </property>
  </action>
  <action name="menu_Filter_Antialiasing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>アンチエイリアシング</string>
   </property>
  </action>
  <action name="menu_View_ScrollHand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>スクロールハンドドラッグ</string>
   </property>
  </action>
  <action name="menu_Window_Thumbnail">
   <property name="text">
    <string>サムネイル</string>
   </property>
  </action>
  <action name="menu_View_FitImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>画像に合わせて表示する</string>
   </property>
  </action>
  <action name="menu_Filter_Binarized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>二値化</string>
   </property>
  </action>
  <action name="menu_Filter_Averagin">
   <property name="text">
    <string>平均化</string>
   </property>
  </action>
  <action name="menu_Window_ToneCurve">
   <property name="text">
    <string>トーンカーブ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
